xml/System/Type.xml

```diff
@@ -13424,9 +13446,16 @@ Types that are defined in the .NET Standard are not marked with <xref:System.Ser
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value indicating whether the <see cref="T:System.Type" /> is an array type that can represent a multi-dimensional array or an array with an arbitrary lower bound.</summary>
+        <value><see langword="true" /> if the current <see cref="T:System.Type" /> is an array type that can represent a multi-dimensional array or an array with an arbitrary lower bound; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the current <see cref="T:System.Type" /> is an array type that can represent a multi-dimensional array or an array with an arbitrary lower bound; otherwise, <see langword="false" />.</value>
```